bitnami/cassandra/README.md

```diff
@@ -144,17 +144,25 @@ The following table lists the configurable parameters of the Cassandra chart and
+| `service.nodePorts.metrics`            | Kubernetes Metrics node port                                                                                            | `""`                                                         |
+| `service.loadBalancerIP`               | LoadBalancerIP if service type is `LoadBalancer`                                                                        | `nil`                                                        |
+| `service.annotations`                  | Annotations for the service                                                                                             | {}                                                           |
+| `networkPolicy.enabled`                | Enable NetworkPolicy                                                                                                    | `false`                                                      |
+| `networkPolicy.allowExternal`          | Don't require client label for connections                                                                              | `true`                                                       |
+| `host.hostNetwork`                     | Use Host-Network for the PODs (if true, also dnsPolicy: ClusterFirstWithHostNet is set)                                 | `false`                                                      |
+| `host.ports.cassandra.intraPort`       | Intra Port on the Host and Container                                                                                    | `7000`                                                       |
+| `host.ports.cassandra.tlsPort`         | TLS Port on the Host and Container                                                                                      | `7001`                                                       |
+| `host.ports.cassandra.jmxPort`         | JMX Port on the Host and Container                                                                                      | `7199`                                                       |
+| `host.ports.cassandra.cqlPort`         | CQL Port on the Host and Container                                                                                      | `9042`                                                       |
+| `host.ports.cassandra.thriftPort`      | Thrift Port on the Host and Container                                                                                   | `9160`                                                       |
+| `host.ports.metrics.httpPort`          | HTTP Port on the Host and Container                                                                                     | `8080`                                                       |
+| `host.ports.metrics.jmxPort`           | JMX Port on the Host and Container                                                                                      | `5555`                                                       |
```

Comment:
Can we use a single `hostNetwork` boolean parameter and rely on the default container ports? This is what we did in other charts (e.g. node-exporter)?
If you consider essential to customize the container ports, add a `containerPorts` array parameter as we do in other charts, e.g. see:
- https://github.com/bitnami/charts/blob/master/bitnami/keycloak/values.yaml#L200

Reply:
Perfect, i changed it to containerPorts.cassandra.*portname*